README.md

```diff
@@ -1,8 +1,22 @@
-# Metadata Server
+# Off-chain metadata tools project
 
-## Running
+## Components
 
-TODO
+This project consists of a number of components:
+  - token-metadata-creator
+    - Asset/Token metadata creation and validation
+  - metadata-server
+    - Server for retrieving metadata from the storage layer
+  - metadata-store-postgres
+    - Postgres metadata storage layer
+  - metadata-webhook
+    - GitHub webhook that writes metadata to the storage layer
+  - metadata-validator-github
```

Comment:
Is this the name which CF expect? I see that it's using `metadata-validator`, and also the Hydra job name is different.
https://github.com/cardano-foundation/cardano-token-registry/blob/master/.github/workflows/ci.yaml